site/public/css/markdown.css

```diff
@@ -90,6 +92,89 @@
     padding: 3px;
 }
 
+/* Styling of markdown input space (MarkdownArea.twig)*/
+.markdown-area {
+    border-radius: 6px 6px 6px 6px;
+    border: 1px solid var(--standard-light-gray);
+}
+
+/* header (.markdown-area-header + children)*/
+.markdown-area-header {
+    display: flex;
+    flex-direction: row;
+    background-color: var(--standard-pale-gray);
+    padding: 0 10px 0 10px;
+    border-radius: 6px 6px 0 0;
```

Comment:
I think I can shorten the padding, but I'm not sure if I can shorted the border-radius since according to [this](https://www.w3schools.com/cssref/css3_pr_border-radius.asp) there seems to be no way to style both the top left and top right corners without 4 values.